engine/class_modules/sc_death_knight.cpp

```diff
@@ -1113,6 +1115,9 @@ inline death_knight_td_t::death_knight_td_t( player_t* target, death_knight_t* p
 
   debuff.biting_cold       = make_buff( *this, "biting_cold", p -> find_spell( 337989 ) )
                            -> set_default_value( p -> conduits.biting_cold.percent() );
+
+  debuff.unholy_blight     = make_buff( *this, "unholy_blight", p -> find_spell( 115994 ) )
+                           -> set_default_value( p -> find_spell( 115994 ) -> effectN( 2 ).percent() );
```

Comment:
You can use ``set_default_value_from_effect`` to avoid repeating ``p -> find_spell( x )``
In this case, this would be ``set_default_value_from_effect( 2 )``
It's only a recent addition which is why it's not everywhere in the module yet

Reply:
You can also fetch the spell from ``p -> talent.unholy_blight -> effectN( 1 ).trigger()``` rather than finding the spell by ID.